components/slack/actions/send-message-private-channel/send-message-private-channel.mjs

```diff
@@ -33,27 +33,6 @@ export default {
         "mrkdwn",
       ],
     },
-    username: {
-      propDefinition: [
-        common.props.slack,
-        "username",
-      ],
-      description: "Optionally customize your bot's username (default is `Pipedream`).",
-    },
-    icon_emoji: {
-      propDefinition: [
-        common.props.slack,
-        "icon_emoji",
-      ],
-      description: "Optionally use an emoji as the bot icon for this message (e.g., `:fire:`). This value overrides `icon_url` if both are provided.",
-    },
-    icon_url: {
-      propDefinition: [
-        common.props.slack,
-        "icon_url",
-      ],
-      description: "Optionally provide an image URL to use as the bot icon for this message.",
-    },
     ...common.props,
```

Comment:
the above removed properties are splatted in here, overwriting the definitions above